src/libraries/Microsoft.PowerFx.Core/Public/Values/ErrorValue.cs

```diff
@@ -77,6 +78,12 @@ public override void ToExpression(StringBuilder sb, FormulaValueSerializerSettin
         {            
             var flag = true;
 
+            if (settings.Compact && _errors.Count == 1)
+            {
+                sb.Append(this);
+                return;
+            }
+
             sb.Append("Error(Table(");
```

Comment:
>Table( [](http://example.com/codeflow?start=29&length=6)

We can skip emitting "Table(" if _errors.Count == 1

Reply:
The tests expect the short form: 
Error(Kind=InvalidArgument)

How are they passing without this?